.github/workflows/ci.yml

```diff
@@ -42,25 +43,18 @@ jobs:
             sleep 1
           done
 
-      - name: Get composer cache directory
-        id: composer-cache
-        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
-
-      - name: Cache dependencies
-        uses: actions/cache@v2
-        with:
-          path: ${{ steps.composer-cache.outputs.dir }}
-          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
-          restore-keys: ${{ runner.os }}-composer-
-
       - name: Install dependencies
-        run: composer install --prefer-dist --no-progress
+        run: |
+          if [ "${SF_VERSION}" = "5.4" ]; then
+            COMPOSER_OPTION="--prefer-lowest"
```

Comment:
Instead of setting a `SF_VERSION` env var, we could just have a "low deps" matrix... because that's really what `5.4` right now means). The config is a bit different, but here's an example: https://github.com/symfony/flex/blob/1.x/.github/workflows/ci.yaml